socorro/tests/lib/test_search_common.py

```diff
@@ -22,7 +22,6 @@
         "data_validation_type": "str",
         "query_type": "string",
         "namespace": "processed_crash",
-        "permissions_needed": [],
```

Comment:
We don't specify these in super search fields anymore. They're specified in the processed crash schema.